pkgs/by-name/an/angular-language-server/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchYarnDeps,
+  fetchFromGitHub,
+}:
+
+mkYarnPackage rec {
```

Comment:
`mkYarnPackage` (AKA `yarn2nix`) was also deprecated since the last release (So the stable manual was not updated to reflect that): 

https://nixos.org/manual/nixpkgs/unstable/#javascript-yarn2nix

Thankfully there should not be hard to change to the new yarn hooks.

Reply:
Oops, already fixed, while I typed :)

Reply:
I was working on a rework to this while you were reviewing. I've switch to a `stdenv.mkDerivation` with the yarn hooks in my latest commit.